packages/api-client/src/types/environment.types.d.ts

```diff
@@ -67,6 +67,19 @@ export interface GetAllEnvironmentsOfProjectResponse {
       profilePictureUrl: string | null
     }
   }[]
+  metadata: {
```

Comment:
Okay for this, I think you can do something like this:

- Move out the main page structure to `index.types.d.ts`:
```typescript
interface Page <T> {
    items: T[]
    metadata: {
        page: number
        perPage: number
        pageCount: number
        totalCount: number
        links: {
        self: string
        first: string
        previous: string | null
        next: string | null
        last: string
    }
    }
}
```
- Use the `Page` type in `GetAllEnvironmentsOfProjectResponse` like this:
```typescript
interface GetAllEnvironmentsOfProjectResponse extends Page <{
    id: string
    name: string
    description: string | null
    createdAt: string
    updatedAt: string
    lastUpdatedBy: {
      id: string
      name: string
      email: string
      profilePictureUrl: string | null
    }
}> {}
 ```

Reply:
Thank you for this.